javascript/javascript1/week3/Series_time/series.js

```diff
@@ -0,0 +1,42 @@
+const seriesDurations = [
+    {
+        title: 'Vikings',
+        days: 2,
+        hours: 16,
+        minutes: 26,
+    },
+    {
+        title: 'The Bridge',
+        days: 1,
+        hours: 14,
+        minutes: 39,
+    },
+    {
+        title: 'Game of thrones',
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+]
+let lifeSpanInMinutes = 80 * 365 * 24 * 60;
+function calculateTotalTime() {
+    let totalMinutes = 0;
+    for (let i = 0; i < seriesDurations.length; i++) {
+        let daysTominutes = seriesDurations[i].days * 24 * 60;
+        let hoursTominutes = seriesDurations[i].hours * 60;
+        let minutes = seriesDurations[i].minutes;
+        let totalMinutesPerSeries = daysTominutes + hoursTominutes + minutes;
```

Comment:
i like this. Variable naming is clean and the calculations are explicit. Small nitpick: `daysToMinutes` and `hoursToMinutes` is a better example of [camelCase](https://en.wikipedia.org/wiki/Camel_case) in variables